app/controllers/api/feedback_submissions_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  class FeedbackSubmissionsController < ApiController
+    def create
+      @feedback = FeedbackSubmission.new(feedback_params)
+
+      if @feedback.save
+        ZendeskTicketsJob.perform_later(@feedback)
+      else
+        render status: 422, json: { messages: @feedback.errors.full_messages }
```

Comment:
Thought more about after we talked, I agree with you on duplicating the validation. I prefer the explicit approach instead of relying on the API spec to proxy the errors, less magic.

I still would like to keep discussing how we are going to ensure API consistency as we start having more for new services